src/app/public/modules/colorpicker/types/colorpicker-color.ts

```diff
@@ -1,6 +1,18 @@
 // spell-checker:ignore Colorpicker,
+/**
+ * Specifies the
+ */
 export interface SkyColorpickerChangeColor {
```

Comment:
Same question as above - re: Public API?

Reply:
Sounds good to me.